cypress/integration/admin_account_test/report_page_layout_test.js

```diff
@@ -0,0 +1,100 @@
+    });
+
+    it("can add a category containing classes",function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("input").should("exist").type("Interactions{enter}",{delay:100});
+            cy.get("li").contains("Interactions").should("exist").click(); //Flaky
+            cy.get('.css-1hwfws3').click().type("Exons{enter}",{delay:100},{force:true});
+            cy.get("ul").filter(".classes").find(".btn-fab").first().click();  //Flaky
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can add description to classes", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click();
+            cy.get("li").contains("Exons").should("exist").click();
+            cy.get(".btn-group-sm").children().first().click();
+
+            cy.get(".classes").within(() => {
+                cy.get("textarea").clear().type("Coding regions within this gene{enter}",{delay:100});
+                cy.contains("Save").click();
+            })
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can edit the visibility of the description of a class", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click()
+            cy.get(".btn-group-sm").children().eq(1).click({delay:100});
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can remove a class from a category", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click();
+            cy.get('.css-1hwfws3').click().type("Introns{enter}",{delay:100},{force:true});
+            cy.get("ul").filter(".classes").find(".btn-fab").last().click();
+            cy.get(".btn-group-sm").eq(1).children().eq(4).click({delay:100});
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    after(function(){
+        cy.visit("/biotestmine/report/MRNA/2000009");
```

Comment:
This hardcoded ID can become invalid in the future. I suggest you use the URL generated in *Copy permanent URL* from its report page instead.

Also, can you move this into a test instead of having it in the after hook? I think those won't be retried, while it seems tests are run in order within a suite, so it should be fine to have it as a regular test.